monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md

```diff
@@ -53,41 +54,41 @@ La palabra Monero es del lenguaje Esperanto. Los creadores decidieron usar Esper
+**El Laboratorio de Investigación de Monero:** Monero no sólo está comprometido en hacer una moneda fungible, sino también en la continua investigación en el campo de la privacidad financiera que involucra a las criptomonedas en general. Con ese fin, investigadores académicos en campos de matemáticas, física, seguridad y computación en blockchain han publicado **[libros blancos](https://lab.getmonero.org/)** y tienen muchas otras **[metas de investigación](https://getmonero.org/es/design-goals/)** en las que están trabajando.
 
 **La minería es accesible:** Cualquiera que pueda conectarse a un dispositivo o navegador web puede
 participar.
 
 ### IMPLICACIONES Y USOS EN LA VIDA REAL
 
-Dado que Monero es seguro, de comisiones bajas y sin fronteras, las personas pueden enviar dinero a pesar de estados o bancos corruptos y/o quebrados. Esto empodera económicamente al individuo de **[países opresores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** o economías deprimidas.
+Dado que Monero es seguro, de comisiones bajas y sin fronteras, las personas pueden enviar dinero a pesar de estados o bancos corruptos y/o quebrados, y se pueden conducir negocios sin competidores husmeando información crítica. Esto empodera económicamente al individuo y negocios en **[países opresores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/),** economías deprimidas o ambientes de negocio altamente competitivos.
 
 El historial financiero privado protege a los consumidores y las compañías de la manipulación de precios, explotación de la cadena de suministros, discriminación economica o algo similar. **[Monero es la única criptomoneda](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)** que tiene las características de servir como dinero electrónico completamente fungible descentralizado.
 
 ### FUNDAMENTOS TÉCNICOS
 
-_(A partir de 11/14/2018)_
-+ **Total de Nodos activos:** 1,756 (Source: https://monerohash.com/nodes-distribution.html)
-+ **Tasa hash de la red:** 435.17 MH/s
-+ **Promedio de Transacciones/Hora:** 178 (30-day average)
-+ **CPU centrales protegiendo la red:** 14,499,178
-+ **Monero en circulación:** 16,570,126 XMR (Approximate)
-+ **Capitalización de mercado:** $1,503,928,296 USD (~0.77% capitalización del mercado de criptomonedas)
-+ **Actual recompensa por bloque:** 3.5984 XMR
+_(A partir del 1/Feb/2018)_
++ **Total de Nodos activos:** 1,564 (Source: https://monerohash.com/nodes-distribution.html)
++ **Tasa hash de la red:** 663.2 MH/s
++ **Promedio de Transacciones/Hora:** 149 (30-day average)
++ **CPU centrales protegiendo la red:** 22,234,297
++ **Monero en circulación:** 16,758,217 XMR (Approximate)
++ **Capitalización de mercado:** $743,944,945 USD (~0.77% capitalización del mercado de criptomonedas)
```

Comment:
We did some changes on this line due to information gathered and noticed by @el00ruobuob in #99, take a look at the comments if you like. Let me add the string with the information we found. 

**Capitalización de mercado:** $732,932,071 USD (~0.65% capitalización del mercado de criptomonedas)

Reply:
Got it, will change it in the next push.